tests/test_logs/test_logs.py

```diff
@@ -480,11 +478,11 @@ def test_describe_log_groups_paging():
 
     resp = client.describe_log_groups(limit=2)
     resp["logGroups"].should.have.length_of(2)
-    resp["nextToken"].should.equal("/aws/lambda/FileMonitoring")
```

Comment:
The `nextToken` value here is just the `logGroupName` when using a real AWS backend.  I don't know if it matters for moto or not... pagination will still work, but the implementation will not match AWS.  I've given some thought to a truly generic paginator for moto, one that does all the heavy lifting but defers some implementation details to the individual services.  I can't say I'm close to submitting anything for review, but it should be possible, which would be nice.  What you're doing here in pulling this out into the utilities module is certainly a start.

Reply:
That's annoying - I assumed the `logGroupName`was just used as token because it's the easiest implementation.

I suppose the easy way would be to extend the pagination_token model with new `hash_value`/`is_hashed` configuration options, but that's not a pretty solution.